src/System.IO.Ports/tests/SerialPort/ReadExisting_Generic.cs

```diff
@@ -113,26 +113,30 @@ public void SuccessiveReadTimeoutNoData()
             }
         }
 
+        [KnownFailure]
         [ConditionalFact(nameof(HasNullModem))]
         public void DefaultParityReplaceByte()
         {
             VerifyParityReplaceByte(-1, numRndChar - 2);
         }
 
+        [KnownFailure]
         [ConditionalFact(nameof(HasNullModem))]
         public void NoParityReplaceByte()
         {
             Random rndGen = new Random(-55);
             VerifyParityReplaceByte('\0', rndGen.Next(0, numRndChar - 1));
         }
 
+        [KnownFailure]
         [ConditionalFact(nameof(HasNullModem))]
         public void RNDParityReplaceByte()
         {
             Random rndGen = new Random(-55);
             VerifyParityReplaceByte(rndGen.Next(0, 128), 0);
         }
 
+        [KnownFailure]
```

Comment:
There are lots of tests that appear to be disabled in this PR.  Why?

Reply:
They are not disabled but they don't run on CI anywhere as it doesn't have any serial devices connected. They were previously not passing and still are not - this is to make it easier to filter those test cases out.

Reply:
most of the issues are related to ParityReplace and XonXoff mode which don't work (I've mentioned all issues I'm aware of in the description minus the ones mentioned in this PR which I'll later add to the list if I don't resolve it)